crates/typst-syntax/src/kind.rs

```diff
@@ -477,7 +477,7 @@ impl SyntaxKind {
             Self::LetBinding => "`let` expression",
             Self::SetRule => "`set` expression",
             Self::ShowRule => "`show` expression",
-            Self::Contextual => "`context` expression",
+            Self::Contextual => "context expression",
```

Comment:
I think the backticks are necessary here, since others have it. Also, if the backticks are absent, I think the codebase refers to it as "context_ual_ expression" :)

Reply:
I also thing it would be more "logical", but the documentation already spells "set rule" and "show rule" without backticks, so I did the same here.

Reply:
I reverted "context" backticks back to what they were before.